neo/Prompt/Utils.py

```diff
@@ -185,7 +185,6 @@ def parse_param(p, wallet=None, ignore_int=False, prefer_hex=True):
             # now it should be unhexxed no matter what, and we can hex it
             return val.hex().encode('utf-8')
 
-        return val
```

Comment:
If we don't remove the value here, should we perhaps also raise an Exception? But then we have an exception handler right afterwards that swallows it, so not sure if necessary.

Reply:
I actually found in further testing that if we don't check for boolean types then some things can get messed up ( `False` parsed as `True`).  So I think we'll need to add a check for `if isInstance(val, bool)`

Reply:
I have updated PR and changed order of conditions and use only `type(val) is not float` to minimize risks.